cli/connector/integration_test.go

```diff
@@ -328,25 +328,28 @@ func runTestMain(m *testing.M) int {
 		WorkDir:      workDir,
 		User:         opts.User,
 		Pass:         opts.Pass,
-		WaitStart:    100 * time.Millisecond,
-		ConnectRetry: 3,
+		WaitStart:    time.Second,
+		ConnectRetry: 5,
 		RetryTimeout: 500 * time.Millisecond,
 	})
 	defer test_helpers.StopTarantoolWithCleanup(inst)
 	if err != nil {
-		log.Fatalf("Failed to prepare test tarantool: %s", err)
+		fmt.Println("Failed to prepare test tarantool:", err)
```

Comment:
Let's replace this err check + print + return code with single `require.NoError` call.

Reply:
`require.NoError` wants `testing.T`, but we have `testing.M` here on setup stage.